docs/docs/ecommerce-tutorial/index.md

```diff
@@ -0,0 +1,551 @@
+
+Serverless is a CLI tool that helps speed up the development of serverless functions and connects the dots between writing the code and making the necessary configurations on different hosting providers.
+
+You can read more about the configurations you’ll use in the `serverless.yml` file in the repo you just cloned or in the Serverless docs. 
+
+`Serverless.yml`
+```
+# Welcome to serverless. Read the docs
+# https://serverless.com/framework/docs/
+
+# Serverless.yml is the configuration the CLI
+# uses to deploy your code to your provider of choice
+
+# The `service` block is the name of the service
+service: gatsby-serverless-stripe
+
+# Configuration variables
+custom:
+ secrets: ${file(secrets.json)}
+
+# The `provider` block defines where your service will be deployed
+provider:
+ name: aws
+ runtime: nodejs6.10
+ stage: dev
+ environment:
+   STRIPE_SECRET_KEY: ${self:custom.secrets.stripeSecretKey} # Stripe secret API key
+
+# The `functions` block defines what code to deploy
+functions:
+ checkout:
+   handler: checkout.handler
+   # The `events` block defines how to trigger the handler.createCharge code
+   events:
+     - http:
+         path: checkout
+         method: post
+         cors: true
+
+```
+
+## Configure serverless with your AWS credentials
+
+Configure serverless with your AWS credentials so you can make updates on AWS through the serverless tools with this command: 
+
+> **NOTE**: It’s helpful to edit this command somewhere other than the terminal as you replace the dummy keys with your real keys, or else it might run halfway through your editing process
+ 
+```shell
+serverless config credentials --provider aws --key AAAAAAAEXAMPLE --secret aaaaaaaa/BBBBBBB/CCCdddEXAMPLEKEY
+```
+
+Where `AAAAAAAEXAMPLE` is your key and `aaaaaaaa/BBBBBBB/CCCdddEXAMPLEKEY` your secret.
+
+If you have trouble finding your way around AWS to get your keys, you can follow these steps taken from the Serverless AWS Credentials docs:
+
+* Create or login to your Amazon Web Services Account and go to the Identity & Access Management (IAM) page.
+* Click on Users and then Add user. Enter a name in the first field to remind you this User is the Framework, like serverless-admin. Enable Programmatic access by clicking the checkbox. Click Next to go through to the Permissions page. Click on Attach existing policies directly. Search for and select AdministratorAccess then click Next: Review. Check everything looks good and click Create user. Later, you can create different IAM Users for different apps and different stages of those apps. That is, if you use the same AWS account for stages/apps, which is more common than using separate accounts.
+* View and copy the API Key & Secret to a temporary place and place them into the command at the beginning of this section of the tutorial before running the command.
```

Comment:
This says _these steps taken from the Serverless AWS Credentials docs_. Maybe this section could be replaced with a link to the relevant AWS docs?